source/deployment/on-boarding.rst

```diff
@@ -1,44 +1,73 @@
 
- - The Support Email is used on email notifications and during tutorial for users to ask support questions.
- - The Help Link is on the Mattermost login page, sign-up pages, and Main Menu. Many organizations use this URL to link to their help desk ticketing system. 
+ - Users can be per-provisioned prior with migration and bulk loading data processes based on prior collaboration systems.  Please see our `migration guide <https://docs.mattermost.com/administration/migrating.html#migration-guide>`_ and `bulk loading documentation <https://docs.mattermost.com/deployment/bulk-loading.html>`_ for additional details.
+ 
+ - `AD/LDAP authentication <https://docs.mattermost.com/deployment/sso-ldap.html#active-directory-ldap-setup-e10-e20>`_ and `SAML authentication <https://docs.mattermost.com/deployment/sso-saml.html>`_ is available for Enterprise Edition, providing identity management, single sign-on and automatic account provisioning.   
+
+4. Enable integrations and plugins to connect your team's workflows and toolsets into Mattermost. 
 
-Migrating to AD/LDAP or SAML from email-based authentication
-------------------------------------------------------------
+ - To enable integrations such as webhooks, slash commands, OAuth2.0, and bots, to go **System Console > Integrations**. More information on these integrations can be found `here <https://docs.mattermost.com/guides/integration.html>`_. 
+ 
+ - To enable and manage plugins, go to **System Console > Plugins**.  Mattermost offers an `integration marketplace <https://integrations.mattermost.com/>`_ where you can see all available plugins available for upload. 
 
-If you have Enterprise Edition, you can migrate from email authentication to Active Directory/LDAP or to SAML Single Sign-on. To set up Active Directory/LDAP, see :doc:`sso-ldap`. To set up SAML Single Sign-on, see :doc:`sso-saml`.
+If your organization  requires more structure and project management artifacts for the implementation of Mattermost, please see our `implemenatation plan <https://docs.mattermost.com/getting-started/implementation_plan.html>`__ 
 
-After the new authentication method is enabled, existing users cannot use the new method until they go to **Account Settings > Security > Sign-in method** and click **Switch to using AD/LDAP** or **Switch to using SAML Single Sign-on**. After they have switched, they can no longer use their email and password to sign in.
+Important Administration Notes 
+------------------------------
+**DO NOT manipulate the Mattermost database**
+
+  - In particular, DO NOT manually delete data from the database directly. Mattermost is designed as a continuous archive and cannot be supported after manual manipulation.
+  - If you need to permanently delete a team or user, use the `Command Line Tool <../administration/command-line-tools.html>`__.
 
 Common Tasks
 ------------
 
-Creating System Admin account from the command line
+**Creating System Admin account from the command line**
```

Comment:
Please make sure the bullet points for this and the other sections here are aligned so that the formatting looks good.